applications/advanced-concepts/bridging.md

```diff
@@ -0,0 +1,16 @@
+---
+parent: Building and research
+title: Bridging
+description: Two-step translation through an intermediate language
+---
+
+**Bridging** uses an intermediate language to translate between source and target languages.
+If a language pair `(source, target)` is desired, bridging translates twice: `(source, bridge)`, then `(bridge, target)`. 
+The `bridge` language is also called a pivot language. 
+
+English is a common choice for the bridge because many languages have training data with English.
+
+## Applications
+
+- Translation without parallel data
```

Comment:
I would link the `parallel data` article here.
The link is: [https://machinetranslate.org/parallel-data](https://machinetranslate.org/parallel-data)